icons/axis-3d.json

```diff
@@ -2,8 +2,7 @@
   "$schema": "../icon.schema.json",
   "contributors": [
     "lscheibel",
-    "danielbayley",
-    "karsa-mistmere"
+    "danielbayley"
```

Comment:
I don’t recall working on this one at all, either…

Reply:
I took the liberty to also remove you from these ones.

Reply:
> I'm afraid you're probably included in many others as well by mistake

@karsa-mistmere Maybe a few slipped through the net… but I have `push`ed a _lot_ of icons/changes! I’ll just submit another PR to remove if I spot any others going forward…